src/woo_publications/metadata/admin.py

```diff
@@ -26,8 +26,44 @@ class InformationCategoryAdmin(AdminAuditLogMixin, OrderedModelAdmin):
         "show_actions",
         "move_up_down_links",
     )
```

Comment:
it's probably a good idea to also add `archiefnominatie` and `bewaartermijn` to the `list_display`